sunpy/net/dataretriever/sources/noaa.py

```diff
@@ -174,6 +174,8 @@ def search(self, *args, **kwargs):
         extractor1 = '{}/warehouse/{:4d}/SRS/{year:4d}{month:2d}{day:2d}SRS.txt'
         extractor2 = '{}/warehouse/{year:4d}/{}'
         matchdict = self._get_match_dict(*args, **kwargs)
+        # this is needed for fetch
+        self._filestart = matchdict["Time"].start
```

Comment:
This shouldn't really be added to the Client object it should be stored in the response object this method returns or if you fetch a different response it wont work as expected.

Reply:
yeah its just a quick patch

so should we just store it in the QueryResponse?

Reply:
I *think* you should insert it into `rowdict` on line 186, but I am not 100% sure.